pmml-evaluator/src/main/java/org/jpmml/evaluator/TransformerContext.java

```diff
@@ -66,6 +61,11 @@ private FieldValue declareMissing(FieldName name) {
         return declare(name, (FieldValue) null);
     }
 
+    @Override
+    protected FieldValue prepare(FieldName name, Object value) {
+        throw new NotImplementedException();
```

Comment:
Should be `java.lang.UnsupportedOperationException` instead.

The `org.jpmml.evaluator.NotImplementedException` has a different meaning - "the PMML specification defines some feature/nehaviour, but this is not yet implemented in the JPMML-Evaluator library"